onnxscript/function_libs/torch_aten/graph_building.py

```diff
@@ -0,0 +1,446 @@
+"""Graph building functions for torchscript graph backend."""
+from __future__ import annotations
+
+import collections
+import typing
+import warnings
+from typing import Any, Optional, Sequence, Union
+
+import numpy as np
+import onnx
+import onnx.checker
+import onnx.defs
+import onnx.shape_inference
+import torch
+from beartype import beartype
+from torch.onnx import _type_utils
+
+import onnxscript
+from onnxscript import evaluator
+from onnxscript import tensor as onnxscript_tensor
+
+__all__ = [
+    "TorchScriptTensor",
+    "TorchScriptGraph",
+    "TorchScriptEvaluator",
+]
+
+
+ValidArgumentType = Union["TorchScriptTensor", str, int, float, bool]
```

Comment:
How about lists of tensors? Or, optionals?

Reply:
Good catch

Reply:
Done